doc/release.rst

```diff
@@ -0,0 +1,113 @@
+.. _release_notes:
+
+********************************************************************************
+Release notes
+********************************************************************************
+
+Tarantool versions later than 1.10.0 and 2.0.0 follow the same rules:
+
+*   ``x.y.0`` is an **alpha version**, just a base for adding new features.
+    We use alpha versions internally and don't release them as distributions.
+*   ``x.y.1`` is a **beta version**, where we add most new features.
+    Beta versions are good for trying new features, but not suitable for production usage.
+*   ``x.y.2``, ``x.y.3`` and later are **release versions**, where we fix bugs and polish newly added features.
```

Comment:
Later versions can only be in 1.10. And new features can only be in 2.x series. Maybe we can say something like that:

``x.y.2`` and ``x.y.3`` are **release versions**, where we fix bugs and polish newly added features. The 1.10 series being an LTS one contains bug fixing only and has the greater versions like 1.10.4, 1.10.5, and so on.

Reply:
But we do add new features in 1.10 series.

Reply:
I would say that we can change behaviour in 1.10, when the old one is considered as definitely broken. Release notes also mentions new OS support as features.

Technically this statement says that we fix bugs and polish features, but it does not mean that we do both in all release series.